src/js/popup/popup.js

```diff
@@ -1,802 +1,1350 @@
+            const newsStoryHeroBodyTextContent = newsItemContent.bodyStringArgs
+              ? browser.i18n.getMessage(
+                  newsItemContent.bodyString,
+                  newsItemContent.bodyStringArgs
+                )
+              : browser.i18n.getMessage(newsItemContent.bodyString);
+            newsStoryHeroBody.textContent = newsStoryHeroBodyTextContent;
+            newsStoryDetail.appendChild(newsStoryHeroBody);
+
+            // If the section has a CTA, add it.
+            if (newsItemContent.fullCta) {
+              const newsStoryHeroCTA = document.createElement("a");
+              newsStoryHeroCTA.classList.add("fx-relay-news-story-link");
+
+              // If the URL points towards Relay, choose the correct server
+              if (newsItemContent.fullCtaRelayURL) {
+                newsStoryHeroCTA.href = `${relaySiteOrigin}${newsItemContent.fullCtaHref}`;
+              } else {
+                newsStoryHeroCTA.href = `${newsItemContent.fullCtaHref}`;
+              }
+              
+              // Set GA data if applicable
+              if (newsItemContent.fullCtaEventLabel && newsItemContent.fullCtaEventAction) {
+                newsStoryHeroCTA.setAttribute("data-event-action", newsItemContent.fullCtaEventAction);
+                newsStoryHeroCTA.setAttribute("data-event-label", newsItemContent.fullCtaEventLabel);
+              }
+
+              newsStoryHeroCTA.textContent = browser.i18n.getMessage(newsItemContent.fullCta);
+              newsStoryHeroCTA.addEventListener("click", popup.events.externalClick, false);
+              newsStoryDetail.appendChild(newsStoryHeroCTA);
+            }
+          },
+        },
+        utilities: {
+          initNewsItemCountNotification: async () => {
+            
+            const localStorage = await browser.storage.local.get();
+
+            const unreadNewsItemsCountExists =
+              Object.prototype.hasOwnProperty.call(
+                localStorage,
+                "unreadNewsItemsCount"
+              );
+              
+            const readNewsItemsCountExists =
+              Object.prototype.hasOwnProperty.call(
+                localStorage,
+                "readNewsItemCount"
+              );
+
+            // First-run user: No unread data present
+            if (!unreadNewsItemsCountExists && !readNewsItemsCountExists) {
+              await browser.storage.local.set({
+                unreadNewsItemsCount: state.newsItemsCount,
+                readNewsItemCount: 0,
+              });
+            }
+
+            // FIXME: The total news item count may differ than what is displayed to the user
+            // Example: Three items total but user doesn't have waffle for one news item. 
+            // Regardless - update the unreadNews count to match whatever is in state
+            await browser.storage.local.set({
+              unreadNewsItemsCount: state.newsItemsCount,
+            });
+
```

Comment:
Suggestion: Should we create an issue for this?

Reply:
Fixed in cdd0de9c8fd3edec29206c4307c1100762298602. Was on my TODO list!